Documentation/EyeTracking/EyeTracking_ExamplesOverview.md

```diff
@@ -1,7 +1,8 @@
 # Eye tracking examples in MRTK
 This page covers how to get quickly started with using eye tracking in MRTK by building on our provided [MRTK eye tracking example package](https://github.com/Microsoft/MixedRealityToolkit-Unity/tree/mrtk_release/Assets/MixedRealityToolkit.Examples/Demos/EyeTracking).
 The samples let you experience one of our new magical input capabilities: **Eye tracking**! 
-The demo includes a number of different use cases ranging from implicit eye-based activations to how to seamlessly combine information about what you are looking at with **voice** and **hand** input. 
+
+The demo includes a number of different use cases ranging from implicit eye-based activations, to how to seamlessly combine information about what you are looking at with **voice** and **hand** input. 
```

Comment:
Why would you add a comma between "range from ... to ..."?

Reply:
Natural pause?